### PR TITLE
[Fix]Recovery Traffic Control Config

### PR DESCRIPTION
**Pull request by qingwli**
## Purpose of the pull request

Due to https://github.com/apache/dolphinscheduler/commit/4114cb07f64950f88ffb4aa8a4ecc4815f32d76b#diff-f2f54230647b5ed6920efc032cd61c4dc2acd8f728ad0f43aef1e8fd3f9793ed , dev branch has no Traffic Control Config in the YAML file. I recovery Traffic Control Config config

## Brief change log

 - modified:   docs/docs/en/architecture/configuration.md
  - modified:   docs/docs/zh/architecture/configuration.md
   - modified:   dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/TrafficConfiguration.java
   - modified:   dolphinscheduler-api/src/main/resources/application.yaml

## Verify this pull request

This pull request is already covered by existing tests, such as *(please describe tests)*.

  - *RateLimitInterceptorTest.*
  - *TrafficConfigurationTest.*

### CODE REVIEW

**Comment by qingwli** [comment]
PTAL @kezhenxu94

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/TrafficConfiguration.java [hunk 1]
```diff
@@ -25,15 +25,15 @@
 @Configuration
 public class TrafficConfiguration {
 
-    @Value("${traffic.control.global.switch:false}")
+    @Value("${traffic.control.global-switch:false}")
```
Comment (ruanwenjun) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/TrafficConfiguration.java]: Can we use `@ConfigurationProperties` to simplify this?
```java
@Data
@Configuration
@ConfigurationProperties(prefix = "traffic.control")
public class TrafficConfiguration {

    private boolean globalSwitch;
    private Integer maxGlobalQpsRate = 300;
    private boolean tenantSwitch;
    private Integer defaultTenantQpsRate = 10;
    private Map<String, Integer> customizeTenantQpsRate = new HashMap<>();
}
```

Comment (qingwli) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/TrafficConfiguration.java]: Done

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Review by SbloodyS — APPROVED** [review]
+1

**Comment by qingwli** [comment]
Hi, which milestone will be specified?

**Comment by SbloodyS** [comment]
> Hi, which milestone will be specified?

I just marked it for the next version.

**Comment by qingwli** [comment]
Sure, thanks @SbloodyS

**Pull request merged**